.github/CONTRIBUTOR_GUIDE/CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+
+### Scope of These Documents
+
+These documents cover the coding standards for the following types of files in our project:
+
+- **`install/$AppName-install.sh` Scripts**: These scripts are responsible for the installation of applications.
+- **`ct/$AppName.sh` Scripts**: These scripts handle the creation and updating of containers.
+- **`json/$AppName.json`**: These files store structured data and are used for the website.
+
+Each section provides detailed guidelines on various aspects of coding, including shebang usage, comments, variable naming, function naming, indentation, error handling, command substitution, quoting, script structure, and logging. Additionally, examples are provided to illustrate the application of these standards.
+
+By following the coding standards outlined in this document, we ensure that our scripts and JSON files are of high quality, making our project more robust and easier to manage. Please refer to this guide whenever you create or update scripts and JSON files to maintain a high standard of code quality across the project. 📚🔍
+
+Let's work together to keep our codebase clean, efficient, and maintainable! 💪🚀
+
+
+## Getting Started
+
+Before contributing, please ensure that you have the following setup:
+
+1. **Visual Studio Code** (recommended for script development)
+2. **Recommended VS Code Extensions:**
+   - [Shell Syntax](https://marketplace.visualstudio.com/items?itemName=bmalehorn.shell-syntax)
+   - [ShellCheck](https://marketplace.visualstudio.com/items?itemName=timonwong.shellcheck)
+   - [Shell Format](https://marketplace.visualstudio.com/items?itemName=foxundermoon.shell-format)
+
+### Important Notes
+- Use [AppName.sh](https://github.com/community-scripts/ProxmoxVE/blob/contributor_guide/.github/CONTRIBUTOR_GUIDE/ct/AppName.sh) and [AppName-install.sh](https://github.com/community-scripts/ProxmoxVE/blob/contributor_guide/.github/CONTRIBUTOR_GUIDE/install/AppName-install.sh) as templates when creating new scripts.
+- The call to `community-scripts/ProxmoxVE` should be adjusted to reflect the correct fork URL.
+
+---
+
+# 🚀 The Application Script (ct/AppName.sh)
+
+- You can find all coding standards, as well as the structure for this file [here](https://github.com/community-scripts/ProxmoxVE/blob/contributor_guide/.github/CONTRIBUTOR_GUIDE/ct/AppName.md).
+- These scripts are responsible for container creation, setting the necessary variables and handling the update of the application once installed.
+
+---
+
+# 🛠 The Installation Script (install/AppName-install.sh)
+
+- You can find all coding standards, as well as the structure for this file [here](https://github.com/community-scripts/ProxmoxVE/blob/contributor_guide/.github/CONTRIBUTOR_GUIDE/install/AppName-install.md).
+- These scripts are responsible for the installation of the application.
```

Comment:
Maybe it make sense to extract this to a `structure.md`.
This document can explain
* the folder structure or this repository
* the structure of some key files (e.g. function libraries)
* the structure of an application (which files build an application) <- that's the two paragraphs here

Reply:
I‘m against splitting it up even more. There should be one seperate file for every script type and one general one to not overcomplicated it.